src/machine/uart.go

```diff
@@ -1,11 +1,19 @@
-// +build atmega esp nrf sam sifive stm32 k210 nxp
+// +build atmega esp nrf sam sifive stm32 k210 nxp rp2040
 
 package machine
 
 import "errors"
 
 var errUARTBufferEmpty = errors.New("UART buffer empty")
 
+type UARTParity int
+
+const (
+	ParityNone UARTParity = 0
```

Comment:
Perhaps you can add some documentation indicating that `ParityNone` is by far the most commonly used?

Reply:
Also done.